CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+## 2.0.24
```

Comment:
Let's bump to 3.0.0 as we are dropping py 2.7.

Reply:
done